server/models/chat.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../config/db');
+
+exports.create = ({ body, id_to, id_from }) => db.one(
+  `INSERT INTO message (body, sender_id, recipient_id)
+      VALUES ($1, $2, $3)
+      RETURNING message_id`,
+  [body, id_to, id_from]
+);
+
+exports.read = (account_id) => db.any(
```

Comment:
Let's group these loose messages into conversations according to the opposing ID (maybe an object of array of type message [key: partner id, value: array of messages]). Let's also make sure to order these arrays by timestamp.